src/helpers/verifyArgs.js

```diff
@@ -0,0 +1,50 @@
+import fs from 'fs';
+import path from 'path';
+
+const verifyArgs = (args) => {
+  if (!args.input) {
+    return createErrorResult('You have to specify input file.');
```

Comment:
It seem a bit over the top, let's use standard javascript Error objects like so:
```javascript
throw new Error('Error message');
```
If we want to be more specific we could build our own error objects based on that one, but I guess standard one would be enough for now. And wrap the invocation of the function in a try catch.

Reply:
Ok, got it.